.gitignore

```diff
@@ -0,0 +1,3 @@
+test.md
+node_modules/
+app_screenshot.png
```

Comment:
- [ ] [OPTIONAL] - You can remove ignoring the .png file from the .gitignore file, it is not required on this milestone. I think, you can just delete this file from your repo, if you will not use it 👍